guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,673 @@
+{
```

Comment:
<p>Running this code connected to an ArcGIS Online org from an environment built using the latest changes to the source code as of 7.28.83, I get an error:</p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[5], line 1
----&gt; 1 site1 = hub_basic.sites.add('Python Site from API')
      2 site1.item

File ~/Job/repos/geosaurus/src/arcgis/apps/hub/sites.py:1053, in SiteManager.add(self, title, subdomain)
   1050     _site_data = json.load(f)
   1052 # Register site and update its data
-&gt; 1053 _data = self._create_and_register_site(
   1054     site, subdomain, _site_data, content_group_id, collab_group_id
   1055 )
   1056 _data = json.dumps(_data)
   1057 site.update(item_properties={"text": _data, "url": domain})

File ~/Job/repos/geosaurus/src/arcgis/apps/hub/sites.py:786, in SiteManager._create_and_register_site(self, site, subdomain, site_data, content_group_id, collab_group_id)
    782     except:
    783         pass
    784 site_data["values"]["layout"]["header"]["component"]["settings"][
    785     "title"
--&gt; 786 ] = title
    787 site_data["values"]["map"] = self._gis.properties["defaultBasemap"]
    788 site_data["values"]["defaultExtent"] = self._gis.properties["defaultExtent"]

NameError: name 'title' is not defined

</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>This is fixed as of PR 10255.</p>